modules/acs-requirements.adoc

```diff
@@ -1,32 +1,16 @@
 // Module included in the following assemblies:
 //
-// * installing/prerequisites.adoc
+// * installing/acs-requirements.adoc
 :_module-type: CONCEPT
 [id="acs-general-requirements_{context}"]
```

Comment:
```suggestion
[id="acs-requirements_{context}"]
```

Reply:
ID needs to mach the module filename (see [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata)).